src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -89,6 +90,34 @@ module sonic-portchannel {
 
 		} /* end of container PORTCHANNEL */
 
+		container PORTCHANNEL_MEMBER {
+
+			description "PORTCHANNEL_MEMBER part of config_db.json";
+
+			list PORTCHANNEL_MEMBER_LIST {
+
+				key "portchannel_name port";
+
+				ext:key-regex-configdb-to-yang "^(PortChannel[0-9]{1,4})|(Ethernet[0-9]+)$";
+
+				ext:key-regex-yang-to-configdb "<portchannel_name>|<port>";
```

Comment:
i thought you are going to remove such regex ext?

Reply:
@lguohan : Yes, once PR https://github.com/Azure/sonic-buildimage/pull/6915 is merged, we do not need them, I added here just to make sure that current tests pass. 
Let me put dependency on #6915 and then raise another commit to remove them. Thx.